jbpm-db-scripts/src/main/resources/db/ddl-scripts/sybase/sybase-jbpm-schema.sql

```diff
@@ -110,7 +110,7 @@
 
     create table CorrelationKeyInfo (
         keyId numeric(19,0) identity not null,
-        name varchar(255) null,
+        name varchar(255) not null,
```

Comment:
I guess though null was allowed before, it was not possible to save it null (or we'd be creating a backward incompatibility), @elguardian can you confirm?